src/plugins/saved_objects/public/plugin.ts

```diff
@@ -19,6 +19,8 @@
 
 import { Plugin } from 'src/core/public';
 
+import './_index.scss';
```

Comment:
I think the pattern now is that if the SASS file is imported only and directly in a JS file, it should no longer use the `_` pattern. So the file name and this import should change to `index.scss`